Source/Dafny/Compilers/Compiler-js.cs

```diff
@@ -2401,6 +2404,17 @@ protected class ClassWriter : IClassWriter {
         }
         nodeProcess.StandardInput.Flush();
         nodeProcess.StandardInput.Close();
+        // Fixes a problem of Node on Windows, where Node does not prints to the parent console its standard outputs.
+        Task.Run(() => {
```

Comment:
Try to avoid fire-and-forget tasks. Errors happening inside the task will be lost.
Either add an exception handler that prints to the desired output, add a continuation responsible to handle errors, or best: [Wait](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.wait?view=net-6.0) for the task's completion so potential errors are reported back to the main thread. Mind that exceptions that occured inside a task are wrapped by an [AggregateException](https://docs.microsoft.com/en-us/dotnet/api/system.aggregateexception?view=net-6.0) that you may want to unwrap first.

Reply:
I wait for the result of this task now just after waiting for the process to exit. I will not handle errors at this point because I don't see a use case for that yet.